### PR TITLE
[WIP] AEROGEAR-3038 - site refresh announcement

### PR DESCRIPTION
**Pull request by jasonmadigan**
WIP.

The post-date will need to be updated to our launch date prior to merging, as "future" posts will not be rendered.

screen shot 2018-06-14 at 12 49 25

### CODE REVIEW

**Comment by jasonmadigan** [comment]
WIP - we'll want to work collaboratively on this one

File: _posts/2018-06-14-aerogear-mobile-services-announcement.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+layout: news
+section: news
+title: Announcing AeroGear Mobile Services on OpenShift
+author: jasonmadigan
+---
+
+Over the past six months, the AeroGear team has been busy building on a new vision for building mobile applications atop of OpenShift. Up until now, the AeroGear project has primarily focused on Push Notifications; we're expanding the suite of services we offer today to include a number of other Mobile Services for use in your applications on OpenShift.
```
Comment (david-martin) [inline _posts/2018-06-14-aerogear-mobile-services-announcement.md]: double use of 'building' reads odd. 'building on a new vision for building'

maybe 'building on a new vision for creating'

File: _posts/2018-06-14-aerogear-mobile-services-announcement.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+AeroGear provides a suite of Mobile Services across a range of native (iOS & Android) and hybrid client SDKs (including Xamarin & Cordova) that will simplify your mobile and modern application development needs, leveraging OpenShift's Container technology to provide a secure, scalable backend platform.
+
+We're really excited in what we've been building, and would encourage you to get started with AeroGear today.
```
Comment (david-martin) [inline _posts/2018-06-14-aerogear-mobile-services-announcement.md]: Should we have a link to a get started page?

File: _posts/2018-06-14-aerogear-mobile-services-announcement.md (hunk 1 above)
Comment (finp) [reply _posts/2018-06-14-aerogear-mobile-services-announcement.md]: maybe 
 team has been busy with a new vision for building mobile applications on OpenShift

Comment (dimitraz) [reply _posts/2018-06-14-aerogear-mobile-services-announcement.md]: working on a new vision?

File: _posts/2018-06-14-aerogear-mobile-services-announcement.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+layout: news
+section: news
+title: Announcing AeroGear Mobile Services on OpenShift
+author: aerogear
+---
+
+Over the past six months, the AeroGear team has been busy with a new vision for building mobile applications on OpenShift. Up until now, the AeroGear project has primarily focused on Push Notifications; we're expanding the suite of services we offer today to include a number of other Mobile Services for use in your applications on OpenShift.
```
Comment (laurafitzgerald) [inline _posts/2018-06-14-aerogear-mobile-services-announcement.md]: Suggestion: Up until now -> Until now

File: _posts/2018-06-14-aerogear-mobile-services-announcement.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+AeroGear provides a suite of Mobile Services across a range of native (iOS & Android) and hybrid client SDKs (including Xamarin & Cordova) that will simplify your mobile and modern application development needs, leveraging OpenShift's Container technology to provide a secure, scalable backend platform.
+
+We're really excited in what we've been building, and would encourage you to [get started](/getting-started/overview/) with AeroGear today.
```
Comment (laurafitzgerald) [inline _posts/2018-06-14-aerogear-mobile-services-announcement.md]: Suggestion: in -> about

**Comment by jasonmadigan** [comment]
Going to merge in as a first pass. I think we'll want to add more detail, though - and there's a date update we'll need to make for the 4th (which I'll handle)

**Pull request merged**